### PR TITLE
Sentinel intro scenario

### PR DESCRIPTION
**Pull request by tr0njavolta**
Sentinel in 60 seconds

### CODE REVIEW

File: learn-sentinel-intro/finish.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Congratulations!
+You've successfully run an infrastructure governance policy using the Sentinel CLI.
+
+[Sign up for free a 30 day trial](https://www.hashicorp.com/blog/announcing-free-trials-for-hashicorp-terraform-cloud-paid-offerings/) of Terraform Cloud **Team & Governance**
+
+## Further reading
+
```
Comment (judithpatudith) [inline learn-sentinel-intro/finish.md:7]: Add a CTA to return to Learn

File: learn-sentinel-intro/step1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+In this scenario, you will apply the Sentinel Policy-as-Code principles to a Terraform specific deployment. You will create a policy that requires your configuration to have specific tags on your S3 buckets and restrict the level of access to your bucket objects.
```
Comment (judithpatudith) [inline learn-sentinel-intro/step1.md:1]:
```suggestion
In this scenario, you will apply Sentinel Policy-as-Code to a Terraform specific deployment. You will create a policy that requires your configuration to have specific tags on S3 buckets and restrict the level of access to bucket objects.
```

File: learn-sentinel-intro/step1.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Open the file `terraform-sentinel/tf-config/main.tf`{{open}} and review the configuration you are testing.
```
Comment (judithpatudith) [inline learn-sentinel-intro/step1.md:3]:
```suggestion
Open the file `terraform-sentinel/tf-config/main.tf`{{open}} and review the infrastructure configuration you are testing.
```

File: learn-sentinel-intro/step1.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+This configuration builds a publicly-readable S3 bucket with a unique name and deploys an example web app as a bucket object. The `acl` attribute of the `"aws_s3_bucket" "bucket"` resource ensures this web app object is public but the viewer cannot write or edit it.
+
+This policy creates a resource filter for your S3 buckets and a rule that requires that resource to have at least one tag.
```
Comment (judithpatudith) [inline learn-sentinel-intro/step1.md:7]: Is this in the right place?

File: learn-sentinel-intro/step1.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+This configuration builds a publicly-readable S3 bucket with a unique name and deploys an example web app as a bucket object. The `acl` attribute of the `"aws_s3_bucket" "bucket"` resource ensures this web app object is public but the viewer cannot write or edit it.
+
+This policy creates a resource filter for your S3 buckets and a rule that requires that resource to have at least one tag. 
+
+## Review the policy
+
+Open the `terraform-sentinel/restrict-s3-buckets.sentinel`{{open}} file and review the policy for this scenario, which requires you to apply at least one tag to any new or updated S3 bucket.
+
+To see Sentinel policy logic in action, run an `apply` with the `trace` flag in your terminal.
```
Comment (judithpatudith) [inline learn-sentinel-intro/step1.md:13]:
```suggestion
Run an `apply` with the `trace` flag in your terminal to apply the policy against data from the infrastructure configuration you just reviewed.
```

File: learn-sentinel-intro/finish.md (hunk 1 above)
Comment (tr0njavolta) [reply learn-sentinel-intro/finish.md:7]: I'm not sure I grok the acronym.

**Pull request merged**